PPPC UtilityTests/ModelTests/ModelTests.swift

```diff
@@ -32,13 +32,15 @@ import XCTest
 
 class ModelTests: XCTestCase {
 
+    var model = Model()
```

Comment:
I would rather have this in a `setup()` method unless we know that the Model is never being changed in any tests.

Reply:
done